docs/commonly_used_data_types/asset_selection.md

```diff
@@ -16,4 +16,14 @@ asset_selection: {
 }
 ```
 
-NOTE: The machine_type strings must match source_type, NOT source_type_clean.
\ No newline at end of file
+~~NOTE: The machine_type strings must match source_type, NOT source_type_clean.~~
```

Comment:
```suggestion

```
Just delete the Note instead of striking throughing it.  Makes for a less confusing reading exprience, especially if opened as text not in Markdown